package/MDAnalysis/coordinates/DMS.py

```diff
@@ -70,7 +70,9 @@ def get_coordinates(self, cur):
         particles = cur.fetchall()
         return [(p['x'], p['y'], p['z']) for p in particles]
 
-    def get_particle_by_columns(self, cur, columns=['x', 'y', 'z']):
+    def get_particle_by_columns(self, cur, columns=None):
```

Comment:
@mnmelo The mutable default argument check only improves the handling of the mutable arguments, not the arguments themselves. As you said, that is a lot more effort. So Cody "suggests" only, what's in your code already ;-)